spring-boot-admin-server-ui-javamelody/src/main/java/net/bull/javamelody/JavaMelodyService.java

```diff
@@ -0,0 +1,46 @@
+package net.bull.javamelody;
```

Comment:
Sorry that I spotted the foreign package declaration this late, but we can't include classes in foreign packages. It must be a `de.codecentric` package.

The downside seems that you cannot access the `net.bull.javamelody.Parameters. So either you ask the javamelody guys to add a java-api to register applications in java melody or you need to make a post request to javameldoy's `CollectorServlet`.